@khanning looks like there are a few lint errors to fix, then I'll review:

```
  358:1   error    Expected indentation of 24 spaces but found 22
  359:1   error    Expected indentation of 28 spaces but found 26
  360:1   error    Expected indentation of 28 spaces but found 26
  361:1   error    Expected indentation of 28 spaces but found 26
  362:1   error    Expected indentation of 24 spaces but found 22
  490:5   error    Missing JSDoc for parameter 'args'                      
  513:13  error    'reducer' is never reassigned. Use 'const' instead
  513:23  error    Arrow function used ambiguously with a conditional expression  no-confusing-arrow
```

--- a/src/extensions/scratch3_microbit/index.js
+++ b/src/extensions/scratch3_microbit/index.js
@@ -355,11 +355,11 @@ class Scratch3MicroBitBlocks {
                     text: 'when [GESTURE]',
                     blockType: BlockType.HAT,
                     arguments: {
-                      GESTURE: {
-                          type: ArgumentType.STRING,
-                          menu: 'gestures',
-                          defaultValue: 'moved'
-                      }
+                        GESTURE: {
+                            type: ArgumentType.STRING,
+                            menu: 'gestures',
+                            defaultValue: 'moved'
+                        }
                     }
                 },
                 '---',
@@ -489,6 +489,7 @@ class Scratch3MicroBitBlocks {
 
     /**
      * Test whether the micro:bit is moving
+     * @param {object} args - the block's arguments.
      * @return {boolean} - true if the micro:bit is moving.
      */
     whenGesture (args) {
@@ -510,8 +511,10 @@ class Scratch3MicroBitBlocks {
      */
     displaySymbol (args) {
         const symbol = cast.toString(args.MATRIX);
-        let reducer = (accumulator, value, index) =>
-            (value === '0') ? accumulator : accumulator + Math.pow(2, index);
+        const reducer = (accumulator, c, index) => {
+            const value = (c === '0') ? accumulator : accumulator + Math.pow(2, index);
+            return value;
+        };
         const hex = symbol.split('').reduce(reducer, 0);
         if (!hex) return;
         this._device.ledMatrixState[0] = hex & 0x1F;

